### DISCUSSION TITLE
Toggle On/Off Zoom level using Scroll

### DISCUSSION THREAD

**Discussion by sukmaharekrsna** [dineug/erd-editor#265] [Ideas]
First I love this extension,
Thanks for adding Zoom Feature

I think it would be nice if I can toggle on/off zoom when scroll
because I think user behavior when scroll was scrolling the media/page

**Comment by dineug** [comment]
Thank you for the good point!

I think it's better to do it with Ctrl or Cmd + wheel.

**Comment by sukmaharekrsna** [comment]
Yups that true, And Thanks For implementing new approach with combination "=" "-"

**Comment by Hoss3770** [comment]
Thanks for the extension 
I love it too

I think it will be nice if I can turn the zoom behaviour when scrolling off completely 
And If I can scroll in list of types with my normal scroll wheel 

Thank you for the extension
